plugins/generator-1.19.4/forge-1.19.4/procedures/entity_get_percent_frozen.java.ftl

```diff
@@ -0,0 +1 @@
+(${input$entity}.getPercentFrozen())
```

Comment:
This is missing a type marker, check existing procedure blocks for `/*@xyz*/` markers and apply accordingly

Reply:
It may also make sense to multiply value by 100 if returned value is in 0 to 1 range to make it true percentage